### PR TITLE
Update iface_hotplug sleep time

### PR DESCRIPTION
**Pull request by hsmj1412**
Need more sleep for attachment take effect on ppc

Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

**Comment by hsmj1412** [comment]
@chunfuwen Plz help review...

File: libvirt/tests/src/virtual_network/iface_hotplug.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def create_iface_xml(mac):
                         iface_list.append({'mac': mac,
                                            'iface_xml': iface_xml_obj})
                 # Need sleep here for attachment take effect
-                time.sleep(5)
+                time.sleep(20)
```
Comment (kylazhang) [inline libvirt/tests/src/virtual_network/iface_hotplug.py]: How about wait_for?

**Comment by chunfuwen** [comment]
please use below solution to set one timeout:
       def xx_method():
            xx

        if not utils_misc.wait_for(xx_method, timeout=timeout):
               test.fail(xx)

File: libvirt/tests/src/virtual_network/iface_hotplug.py (hunk 1 above)
Comment (hsmj1412) [reply libvirt/tests/src/virtual_network/iface_hotplug.py]: Done

**Comment by hsmj1412** [comment]
@chunfuwen Done

**Comment by balamuruhans** [comment]
Thanks all, let us get it in.

**Pull request merged**